docs/cloud/tcloud_getting_started.md

```diff
@@ -64,9 +64,13 @@ Technical Requirements:
 
```

Comment:
remove my getting started loom video given it's too divergent from the new setup